buildScript/depends.gincl

```diff
@@ -30,7 +30,7 @@ ext {
               ':websocket-api']
 
   mysql_lib = ':mysql-connector-java-5.1.6-bin'
-  starlink_lib = [':rayreg', ':regclient', ':stil']
+  starlink_lib = [':rayreg', ':starlink-registry', ':jcdf', ':eag-healpix', ':ndx', ':mirage', ':gbin', ':vecmath', ':commons-compress', ':starlink-pal', ':starlink-cdf', ':starlink-connect', ':starlink-fits', ':starlink-table', ':starlink-util', ':starlink-votable']
```

Comment:
If you have the info, it might be a good idea to include the version into the file name.  It's useful in the future when we need to update.